src/main/java/net/fabricmc/loader/api/FabricLoader.java

```diff
@@ -211,6 +211,13 @@ static FabricLoader getInstance() {
 	@Deprecated
 	File getGameDirectory();
 
+	/**
+	 * Get the current directory for temporary files.
+	 *
+	 * @return the cache directory
+	 */
+	Path getCacheDir();
```

Comment:
Would it be better for this to take the modid? To enforce using `.cache/modid`

Reply:
Idea: If this took the modid it would be possible for it to cleanup directories that are no longer used, is this a good idea? Im not too sure, something to think about.

Reply:
I'm not sure if I like automatic cleanup, however encouraging the usage of a separate directory per mod sounds good.